lib/rubocop/cop/rails/orm_abstraction.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Avoid breaking ORM abstraction.
```

Comment:
This should certainly have some bigger rationale here.

Reply:
Yeah, this cop basically requires a book to adequately explain for some developers. What is an abstraction? Why would you make an abstraction? What is an Object-Relational Mapping abstraction? What does this have to do with Rails?

Would a few links be acceptable?